lib/main.js

```diff
@@ -245,6 +261,7 @@ let load = loader({
             try {
               await pollSpotRequests({ec2: ec2[region], region, state});
             } catch (err) {
+              console.dir(err.stack || err);
```

Comment:
I'm not using the debug module, but I guess using `taskcluster-lib-log` would be better here.